prometheus/exporter/gpu_exporter.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/python
+import subprocess
+import sys
+from xml.dom import minidom
+
+def parse_xml(smi, logDir):
+    xmldoc = minidom.parse('test.xml')
+    gpuList = xmldoc.getElementsByTagName('gpu')
+    print(len(gpuList))
+    gpu_count = len(gpuList)
+    print("gpu numbers" + str(gpu_count))
+    nvidiasmi_attached_gpus = "nvidiasmi_attached_gpus" + "{} " + str(gpu_count) 
+    outputFile = open(logDir + "gpu_exporter.prom", "w")
+    outputFile.write(nvidiasmi_attached_gpus + "\n")
+    for gpu in gpuList:
+        minorNumber = gpu.getElementsByTagName('minor_number')[0].childNodes[0].data
+        gpuUtil = gpu.getElementsByTagName('utilization')[0].getElementsByTagName('gpu_util')[0].childNodes[0].data.replace("%", "")
+        gpuMemUtil = gpu.getElementsByTagName('utilization')[0].getElementsByTagName('memory_util')[0].childNodes[0].data.replace("%", "")
+        nvidiasmi_utilization_gpus = "nvidiasmi_utilization_gpus" + "{minor_number=" + str(minorNumber) + "}" + " " + str(gpuUtil)
```

Comment:
>"nvidiasmi_utilization_gpus" + "{minor_number=" + str(minorNumber) + "}" + " " + str(gpuUtil) [](start = 37, length = 93)

Suggest use string.format(), and please define the format of metrics as a common string.